core/chains/evm/log/registrations_unit_test.go

```diff
@@ -54,3 +72,100 @@ func TestRegistrationsUnit_PanicsOnDoubleSubscribe(t *testing.T) {
 	// Now we can add it again
 	r.addSubscriber(sub)
 }
+
+func newTopic() Topic {
```

Comment:
optional: Is it ok to have all non-tests put in a block before all tests, for readability?
I mean all TestUnit_ functions, which are actually executed, should be say, at the end of the file, making readability easier.